chucks-pick3-web/assets/css/style-sheet.css

```diff
@@ -272,12 +272,17 @@ link-container {
 
 .how-to-container {
     display: block;
-    background-color: #a9c0ee;
-    width: 33.33%;
-    height: 300px;
-    padding: 0px;
-    margin: auto;
-    position: center;
+    background-image: url("../img/web icons-01.png");
```

Comment:
Any reason why we are updating the .how-to-container rule?

Reply:
I updated and removed bullets and made width equal for each div